### PR TITLE
[adxl345] Small adxl345 and resonance testing improvments

### PR DESCRIPTION
**Pull request by dmbutyugin**
Signed-off-by: Dmitry Butyugin <dmbutyugin@google.com>

### CODE REVIEW

File: docs/G-Codes.md [hunk 1]
```diff
@@ -704,10 +704,13 @@ The following commands are available when an
   the first time, it starts the measurements, next execution stops
   them. If RATE is not specified, then the default value is used
   (either from `printer.cfg` or `3200` default value). The results of
-  measurements are written to a file named `/tmp/adxl345-<name>.csv`
-  where `<name>` is the optional NAME parameter. If NAME is not
-  specified it defaults to the current time in "YYYYMMDD_HHMMSS"
-  format.
+  measurements are written to a file named
+  `/tmp/adxl345-<chip>-<name>.csv` where `<chip>` is the name of the
```
Comment (KevinOConnor) [inline docs/G-Codes.md:708]: If we change the behavior if this command I think we should note the change in the docs/Config_Changes.md file.

File: docs/Measuring_Resonances.md [hunk 2]
```diff
@@ -51,6 +51,13 @@ Note that on a bed slinger printer one must design 2 mounts: one for the
 toolhead and one for the bed, and run the measurements twice. See the
 corresponding [section](#bed-slinger-printers) for more details.
 
+**Attention:** neither the accelerometer nor the screws that hold it in
+place (if any) should not touch any metal parts of the printer.
```
Comment (KevinOConnor) [inline docs/Measuring_Resonances.md:55]: Double negative?  How about: `make sure the accelerometer and any screws that hold it in
place do not touch any metal parts of the printer.`

File: docs/Measuring_Resonances.md [hunk 3]
```diff
@@ -51,6 +51,13 @@ Note that on a bed slinger printer one must design 2 mounts: one for the
...
+Basically, the mount must be designed such as to ensure the electrical
+isolation of the accelerometer from the printer frame. Failing to ensure
+that can create a ground loop in the system, and damage the electronics
+in the worst case scenario.
```
Comment (KevinOConnor) [inline docs/Measuring_Resonances.md:59]: Heh - I'm sure there's an even worse, "worse case scenario".  How about just `Failing to ensure
that can create a ground loop in the system that may damage the electronics.`

**Review by KevinOConnor — COMMENTED** [review]
Thanks.  I have a couple of minor comments.

-Kevin

File: docs/Measuring_Resonances.md (hunk 2 above)
Comment (dmbutyugin) [reply docs/Measuring_Resonances.md]: Yes, that indeed sounds better.

File: docs/Measuring_Resonances.md (hunk 3 above)
Comment (dmbutyugin) [reply docs/Measuring_Resonances.md]: That is probably true. OK, I've changed the wording per your suggestion.

File: docs/G-Codes.md (hunk 1 above)
Comment (dmbutyugin) [reply docs/G-Codes.md:708]: If it is worth mentioning - sure. I also added the `TEST_RESONANCES` command change there then.

**Comment by dmbutyugin** [comment]
Thanks for the review, Kevin! I tentatively put the today's date to Config_Changes.md.

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**